test/integration/targets/apt/tasks/upgrade.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Remove packages that cause the upgrade below to fail, they need manual intervention to upgrade
+  apt:
+    name: "{{ pkg }}"
+    state: absent
+  with_items:
+    - mysql-server-5.7
+    - mysql-server
+  loop_control:
+    loop_var: pkg
```

Comment:
This may have unintended interactions with other integration tests (most likely the mysql* ones). Once tests are passing for this PR we should run some additional tests to make sure we don't introduce any issues there.

Reply:
Good catch, indeed it broke mysql tests. Fixed it, the following works now:
```
ansible-test integration apt mysql_db mysql_user mysql_variables -v --docker ubuntu1604py3
```